runner.rb

```diff
@@ -4,15 +4,15 @@
 require './lib/game'
 
 computer_data = {
-  "computer_board" => Board.new,
-  "cruiser" => Ship.new("Cruiser", 3),
-  "submarine" => Ship.new("Submarine", 2)
+  computer_board: Board.new,
+  cruiser: Ship.new("Cruiser", 3),
+  submarine: Ship.new("Submarine", 2)
```

Comment:
In future refactoring, I am thinking this will come out of the runner file and into its own class (like "ship_builder.rb" or something).